src/ts/components/settings/SettingsPanelPageOpenButton.ts

```diff
@@ -11,8 +11,8 @@ export class SettingsPanelPageOpenButton extends SettingsPanelPageNavigatorButto
     super(config);
 
     this.config = this.mergeConfig(config, {
-      cssClass: 'ui-settingspanelpageopenbutton',
-      text: i18n.getLocalizer('open'),
+      cssClass: 'ui-textbutton',
```

Comment:
The css class usually mirrors the component name. Please rename the component accordingly as well (and consider the suggestion from the first comment)